src/math/web_mercator.rs

```diff
@@ -0,0 +1,143 @@
+//! Calculations with Web Mercator coordinates and tiles.
+
+use nalgebra::Vector2;
+use nav_types::WGS84;
+use std::f64::consts::{FRAC_1_PI, PI};
+
+/// 2.0 * E.powf(PI).arctan() - FRAC_PI_2;
+/// In degrees, it's 85.051129 (cf. Wikipedia)
+const LAT_BOUND_RAD: f64 = 1.484_422_229_745_332_4;
+
+/// LAT_BOUND_SIN = sin(LAT_BOUND_RAD)
+const LAT_BOUND_SIN: f64 = 0.996_272_076_220_75;
+
+const TWO_PI: f64 = 2.0 * PI;
+const FOUR_PI: f64 = 4.0 * PI;
+const FOUR_PI_INV: f64 = 0.25 * FRAC_1_PI;
```

Comment:
How about renaming this to `FRAC_1_FOUR_PI`?

Reply:
Done.